src/code42cli/date_helper.py

```diff
@@ -15,90 +14,36 @@
 )
 
 
-def verify_timestamp_order(min_timestamp, max_timestamp):
-    if min_timestamp is None or max_timestamp is None:
+def convert_datetime_to_timestamp(dt):
+    if dt is None:
         return
-    if min_timestamp >= max_timestamp:
-        raise click.BadParameter(
-            param_hint=["-b", "--begin"], message="cannot be after --end date."
-        )
+    return (dt - datetime.utcfromtimestamp(0)).total_seconds()
```

Comment:
This is like the python2 compat way of doing this, not that it is invalid.

We can make use of the python3 feature and just do `dt.timestamp()`, I think

Reply:
```
>>> n.timestamp()
1610037626.854844
>>> (n - datetime.utcfromtimestamp(0)).total_seconds()
1610037626.854844
>>> 
```

Reply:
updated.